src/ziggurat/config.clj

```diff
@@ -76,3 +81,16 @@
 (defn channel-retry-config [topic-entity channel]
   (get-in (ziggurat-config) [:stream-router topic-entity :channels channel :retry]))
 
+(defn- construct-java-response [config-vals]
```

Comment:
Why do mean by construct java response?

Reply:
when config is fetched, the function can return a clojure PersistentHashMap, we are converting it to a Java Map and converting keywords to strings.

Reply:
Function name is not explanatory.

Reply:
changed-it